bin/plotting/pycbc_plot_vt_ratio

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+
+import h5py
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+from collections import OrderedDict
+import numpy as np
+import argparse
+
+parser = argparse.ArgumentParser()
+parser = argparse.ArgumentParser(usage="",
+    description="Plot ratios of VTs calculated at various IFARs using pycbc_page_sensitivity for two analyses")
+parser.add_argument('--comp-1-file', required=True,
```

Comment:
suggest to change file names to e.g. 'vt-file-one' and 'vt-file-two' .. numbers appearing in option names make me nervous

Reply:
I agree that option names without numbers would be preferable

Reply:
makes sense, this will be updated